docker_tippecanoe/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.1" 
+services:
+  wbeep_processor_tippecanoe:
+    image: code.chs.usgs.gov:5001/wma/iidd/wbeep-data-processing:tippecanoe-latest
+    build:
```

Comment:
I think here you probably don't need the `dockerfile`key since `Dockerfile` is default

Reply:
Right, I was considering naming the dockerfiles something different but decided to go with directories instead